tests/mocks/components.py

```diff
@@ -317,12 +317,11 @@ def __init__(self, name: str):
         super().__init__(name)
 
     async def get_image(self, mime_type: str = CameraMimeType.PNG, **kwargs) -> Union[Image.Image, RawImage]:
-        if not CameraMimeType.is_supported(mime_type) or mime_type == CameraMimeType.RAW:
+        mime_type, is_lazy = CameraMimeType.from_lazy(mime_type)
+        if is_lazy or (not CameraMimeType.is_supported(mime_type)):
             return RawImage(
                 data=self.image.convert("RGBA").tobytes("raw", "RGBA"),
```

Comment:
[q] the call to convert-- will this end up decoding the image anyway, just to convert to RGBA and then put it back into bytes? 

If that's the case, it defeats the purpose of lazy, which is about not touching the bytes it receives and just passing them along to the next computer that needs it.

Reply:
This is just the test case -- but yes if someone were to do this then it would defeat the purpose of lazy.